src/transformState.js

```diff
@@ -46,7 +46,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  transforms.forEach(({ operation, properties }) => {
+    switch (operation) {
+      case 'addProperties': {
+        return Object.assign(state, properties);
+      }
+
+      case 'removeProperties': {
+        return properties.forEach(property => delete state[property]);
+      }
+
+      case 'clear': {
+        return Object.keys(state).map(key => delete state[key]);
+      }
+
+      default: {
+        return state;
```

Comment:
you do not need to return from forEach, will be better to remove return statements and add breaks